crates/bevy_reflect/derive/src/serialization.rs

```diff
@@ -77,13 +77,14 @@ pub(crate) struct SkippedFieldDef {
 impl SkippedFieldDef {
     pub fn new(field: &StructField<'_>) -> Result<Self, syn::Error> {
         let ty = &field.data.ty;
+        let bevy_reflect_path = BevyManifest::default().get_path("bevy_reflect");
```

Comment:
What kind of impact does this have on compile times? We could also consider passing the `bevy_reflect_path` down (`ReflectDerive::from_input` -> `SerializationDataDef::new` -> `SkippedFieldDef::new`)

Reply:
I don't know about compile time changes, but I didn't like this solution anyway. With that map you've provided I've just threaded the `bevy_reflect_path` the whole way down instead, which should definitely have a negligible impact on compilation now.